members/_posts/2016-10-04-lewic.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: memberpage
+category: cmember
+logo : lewic.png

```

Comment:
Yes we can do that, but it is scaled smaller on member list, I have tested this.